Dockerfile

```diff
@@ -165,20 +169,47 @@ RUN mkdir /home/dangerzone/.containers
 # The `ln` binary, even if you specify it by its full path, cannot run
 # (probably because `ld-linux.so` can't be found). For this reason, we have
 # to create the symlinks beforehand, in a previous build stage. Then, in an
-# empty contianer image (scratch images), we can copy these symlinks and the
-# /usr, and stich everything together.
+# empty container image (scratch images), we can copy these symlinks and the
+# /usr, and stitch everything together.
 ###############################################################################
 
 # Create the filesystem hierarchy that will be used to symlink /usr.
 
-RUN mkdir /new_root
-RUN mkdir /new_root/root /new_root/run /new_root/tmp
-RUN chmod 777 /new_root/tmp
+RUN mkdir -p \
+    /new_root \
+    /new_root/root \
+    /new_root/run \
+    /new_root/tmp \
+    /new_root/home/dangerzone/dangerzone-image/rootfs
+
+# XXX: Remove /etc/resolv.conf, so that the network configuration of the host
```

Comment:
This `XXX` seem to be solved (the remove is done a few lines below)

Reply:
Right, the purpose of `XXX` here is to explain what's the need for removing `/etc/resolv.conf` in the following lines, since it will come as a surprise for most people.

Reply:
Good point. Done in 8ca1357a415d11b844d48dffc3ddb791f68176a1.

Reply:
I'm not sure to get it. The `XXX` is still there. Maybe an error?

Reply:
I changed the comment phrasing to show that removing `resolv.conf` is not a todo item, but something that we intentionally do, for reasons that may not be immediately clear. The `XXX` is there as a signal for our future selves (or a passer-by), to alert them about this unexpected action.

Reply:
Sorry, but then I don't understand why we have it show as a todo list. If it's not one, why not just putting a comment?

Reply:
Could it be that we interpret `XXX` differently? To me, `XXX` means "Alert, we're doing something unexpected here". They are not `FIXME` or `TODO` tags, because it may be something that's just the way it is, albeit surprisingly so.

That's why I'm using it to explain the removal of `resolv.conf`. Do you interpret it the same way, or do you view it as a `TODO` thing?

Reply:
Interesting, that's the bottom of it: it doesn't have the same meaning for you and me. To me, `XXX` is an alias for `TODO` or `FIXME`.

I've read a bit more on the subject, and [this stackoverflow post](https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments) was interesting.

Could replacing the `XXX` with `NOTE` work for you?

Reply:
Sure, I can go with that. For me, `NOTE` indicates something more benign in nature, so I don't tend to use it in lines where changing something may impact security. I prefer to grab the reader's attention with `XXX`.

I do understand though that this preference may not be shared, so I'll try to adjust and use `NOTE` from now on. Check out e715bd6.